### ISSUE TITLE
theme screenshots?

### ISSUE THREAD

**Issue by tjaartvdwalt** [acrisci/i3-style#28]
It would be pretty cool if there was a page showing theme screenshots.

**Comment by acrisci** [comment]
I totally agree.

It would be even cooler if there was a program that could automatically generate them.

**Comment by Swivelgames** [comment]
Definitely. I'm interested in this, but I'm not entirely sold on using it. Screenshots would definitely sell me on what it is that I'm actually installing
